server/models/bill.js

```diff
@@ -10,7 +10,9 @@ const BillSchema = new Schema(
 		date: Date,
 		dishes: [
 			{
-				user: { type: Schema.ObjectId, required: true },
+				// Does user have to be required since we have the userEmail alraedy.
+				user: Schema.ObjectId,
+				// user: { type: Schema.ObjectId, required: true },
```

Comment:
Changed user field in dishes from required to optional. The reason being we have the user email already.